packages/varlock-website/src/content/docs/integrations/cloudflare-workers.mdx

```diff
@@ -1,13 +0,0 @@
----
-title: Cloudflare Workers
-description: How to integrate Varlock with Cloudflare Workers for secure, type-safe environment management
----
-
-[Cloudflare Workers](https://developers.cloudflare.com/workers/) are extremely powerful, but dealing with env vars within them can be challenging.
-Varlock can simplify that, providing validation, type-safety, flexible multi-environment management, and extra security guardrails to ensure you never leak a secret.
-
-If using a framework (like Next.js) deployed on Cloudflare Workers, you can use the integration for that framework.
-
-However, to use varlock in a Cloudflare Workers app without a framework, you must use our [Vite integration](/integrations/vite/), along with the [Cloudflare Vite plugin](https://developers.cloudflare.com/workers/vite-plugin/). If you are not yet using their Vite plugin, it is very easy to set up, and provides some [great DX improvements](https://blog.cloudflare.com/introducing-the-cloudflare-vite-plugin/) to your workflow.
```

Comment:
this is a bit of an important note - that they _must_ use the vite plugin - since the default workflow is not to use it.